.github/workflows/ci.yaml

```diff
@@ -481,7 +481,7 @@ jobs:
   deploy-demo-env:
     name: Deploy updated images to the demo environment
     runs-on: self-hosted
-    if: vars.DEPLOY_DEMO == 'true' && (github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch')
+    if: (vars.DEPLOY_DEMO == 'true' || github.event.action == 'demo-deploy') && (github.event_name == 'release' || (github.event_name == 'push' && github.ref_name == 'main') || github.event_name == 'workflow_dispatch')
```

Comment:
Shouldn't this be an `and`? I mean, otherwise, we won't be able to skip the deployment.
Other option would be to add the `DEPLOY_DEMO` variable to the wanda project as well, so we can disable it from there.
What do you think?

Reply:
I prefer the approach of also adding the variable in wanda's CI. I already added it :+1: